starfish/__init__.py

```diff
@@ -1 +1,9 @@
+# top-level objects
 from .starfish import starfish
+from .intensity_table import IntensityTable
+from .codebook import Codebook
+from .experiment import Experiment
+from ._stack import ImageStack
```

Comment:
food for thought: why is _stack the only hidden module?

Reply:
This is a good question. I'm open to making more or less all of these private. What's your opinion?

Reply:
It makes it hard to import other stuff in those packages, potentially. I don't know how common that is.